cardano-cli/src/Cardano/CLI/Types/Errors/BootstrapWitnessError.hs

```diff
@@ -14,6 +14,9 @@ data BootstrapWitnessError
     MissingNetworkIdOrByronAddressError
   deriving Show
 
+instance Error BootstrapWitnessError where
+  prettyError = renderBootstrapWitnessError
```

Comment:
Let's remove `render*` functions for `Error`s and just use `prettyError`. Could be out of scope for this PR, because it's only slightly related.

Reply:
We can do that in a follow up PR. I agree we should remove `render*` functions and rely solely on the `Error` instances.